src/Cache/Cache.php

```diff
@@ -68,6 +68,7 @@ public function toJson(): string
             'php' => $this->getSignature()->getPhpVersion(),
```

Comment:
would you mind to clarify this one first? https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6496#issuecomment-1196420448